src/Commands/GenerateFeedsCommand.php

```diff
@@ -30,27 +32,22 @@ class GenerateFeedsCommand extends Command
      * @var string
      */
     protected $description = 'Generate the enabled feeds for the enabled stores.';
-    /**
-     * @var FeedsHandler
-     */
-    private $feedsHandler;
+
+    private FeedsHandler $feedsHandler;
 
     public function __construct(FeedsHandler $feedsHandler)
     {
         parent::__construct();
         $this->feedsHandler = $feedsHandler;
     }
 
-
-    /**
-     * @return void
-     * @throws \Throwable
-     */
-    public function handle()
+    public function handle(): void
     {
-        [$feeds, $stores] = $this->feedsHandler->generateFeeds();
-
-
-        $this->info('Queued '. count($feeds) * $stores->count() . ' feeds.');
+        try {
+            [$feeds, $stores] = $this->feedsHandler->generateFeeds();
+            $this->info('Queued '. count($feeds) * $stores->count() . ' feeds.');
+        } catch (Throwable $e) {
+            $this->error($e->getMessage());
```

Comment:
Do you expect the feeds generation to error?

Reply:
Not necessarily, but it throws an exception if it does and I suppose we want to catch (and log) that